motioneye/static/js/main.js

```diff
@@ -2919,15 +2919,23 @@ function runPictureDialog(entries, pos, mediaType) {
     
     var img = $('<img class="picture-dialog-content">');
     content.append(img);
-    
+
+    var video_container = $('<video class="picture-dialog-content" controls="true">');
+    var video_source = $('<source type="video/mp4">')
```

Comment:
Remove this and use `src` directly on `<video>` or pass mime type from backend and insert it here.